### PR TITLE
Adding relnotes for 0.3.1811.203

### PR DESCRIPTION
**Pull request by anpaz-msft**

### CODE REVIEW

File: api/canon/Microsoft.Quantum.Canon.yml [hunk 1]
```diff
@@ -1252,27 +1362,27 @@ operations:
     a register.
 - uid: microsoft.quantum.canon.with
   summary: >-
-    Given operations implementing operators $U$ and $V$, performs the
-    operation $UVU^{\dagger}$ on a target. That is, this operation
-    conjugates $V$ with $U$.
+    Given operations implementing operators `U` and `V`, performs the
+    operation `U�VU` on a target. That is, this operation
```
Comment (cgranade) [inline api/canon/Microsoft.Quantum.Canon.yml]: It looks like the ⁺ got corrupted here somehow?

File: articles/relnotes/index.md [hunk 2]
```diff
@@ -11,6 +11,19 @@ uid: microsoft.quantum.relnotes
...
 
+
+# Version 0.3.1811.203
+
+*Release date: November 2, 2018*
+
+This release includes a few bug fixes, including:
+
+* Invoking `DumpMachine` could change the state of the simulator under certain situations.
+* Removed compilation warnings when building projects using a version of .net core previous to 2.1.403.
```
Comment (cgranade) [inline articles/relnotes/index.md]:
```suggestion
* Removed compilation warnings when building projects using a version of .NET Core previous to 2.1.403.
```

File: articles/relnotes/index.md [hunk 3]
```diff
@@ -11,6 +11,19 @@ uid: microsoft.quantum.relnotes
...
 
 
+
+# Version 0.3.1811.203
+
+*Release date: November 2, 2018*
+
+This release includes a few bug fixes, including:
+
+* Invoking `DumpMachine` could change the state of the simulator under certain situations.
```
Comment (cpalmer2020) [inline articles/relnotes/index.md]: Fixed issue where 'DumpMachine' changed the state of the …"

Comment (cpalmer2020) [reply articles/relnotes/index.md]: Otherwise, it soundsl like you are creating the bug.

**Pull request merged**